### ISSUE TITLE
View is cropped until fullscreen-toggle

### ISSUE THREAD

**Issue by falkecarlsen** [hrjakobsen/slext#62]
Upon loading an Overleaf-project, the file-tree list is cropped. See picture of default AAU project-report structure - I'd expect `master.tex` at the bottom. Note that the `preamble.tex` button is slightly cropped as well.

![image](https://user-images.githubusercontent.com/11318702/57374263-cc6b5800-719a-11e9-8d97-cc3c6392d68c.png)

Upon fullscreening the browser-window, the view resizes correctly and persists when exiting full-screen viewing:

![image](https://user-images.githubusercontent.com/11318702/57374439-3126b280-719b-11e9-853b-a00d18679e7c.png)

It seems that the height of toolbar is exactly the amount of crop that is applied and visually it seems to happen when the toolbar loads.

This is observed on Firefox 66.0.5-1 and back, under i3 and X11, if this is platform-dependent.

**Comment by cogitantium** [comment]
Reproducible under Win10 on Firefox 66.0.4 64-bit as well.

**Comment by hrjakobsen** [comment]
And the list is not scrollable when cropped? I assume this has to do with the old problem from #26, where the fix was to scroll to the top of the main container. I thought I'd fixed this behavior by adding a padding to the file container though. I'll take a look and see what can be done.

**Comment by cogitantium** [comment]
No, the list is scrollable at all times. It seems that the file-tree toolbar is only 32px high, and the main toolbar is 39px high. Each item in the list is 32.8px high, and as the first picture shows, the view is cropped more than a single item's height. So I believe the issues does lie with the main toolbar.

Is the issue not reproducible for you?

**Comment by hrjakobsen** [comment]
![image](https://user-images.githubusercontent.com/3295293/57513974-c1860400-730f-11e9-9ffb-da2d2bdc63cf.png)
I'm able to see all items in the list, with `sources.bib` being the last file. The file is _slightly_ cropped, but nothing that hides an entire element.

After triggering the scroll-to-top fix, i get a larger gap at the bottom:
![image](https://user-images.githubusercontent.com/3295293/57514142-1164cb00-7310-11e9-9afc-fc72da6ee113.png)

... but still nothing that hides a file.

**Comment by cogitantium** [comment]
Hmm. Could it be browser-dependent?

Another idea could be that the structure of the project triggers the issue. Try opening an AAU-report template which has large directories and a single tex-file in root.

**Comment by hrjakobsen** [comment]
![image](https://user-images.githubusercontent.com/3295293/57688874-c237d680-763e-11e9-976f-acef21def551.png)
With the AAU report template i can get it to cover the file slightly, but nothing that hides an entire file.
